src/aspire/estimation/covar2d.py

```diff
@@ -57,6 +56,7 @@ def _get_covar(self, coeffs, mean_coeff=None,  do_refl=True):
             raise RuntimeError('The coefficients need to be calculated first!')
         if mean_coeff is None:
             mean_coeff = self._get_mean(coeffs)
+        mean_coeff = mean_coeff.reshape((self.basis.count, 1))
```

Comment:
This can be avoided by simply replacing `mean_coeff[mask, ...]` below by `mean_coeff[mask, np.newaxis]`.

Reply:
Removed.